src/NLOD-2.0.xml

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="NLOD-2.0"
+            name="Norwegian Licence for Open Government Data">
```

Comment:
Two comments on the `<license>` tag:

* Can you include the version number (2.0) in the title?
* Can you also include a ` listVersionAdded="3.14"` property? See https://github.com/spdx/license-list-XML/blob/master/src/BSD-3-Clause-No-Military-License.xml for an example

Reply:
Done!

Would you like me to update the NLOD-1.0 license definition as well? I notice that one has a "full" name of "Norwegian Licence for Open Government Data (NLOD) 1.0" as well.

Reply:
Thanks @evancharlton!

Yes, if you want to submit an update for NLOD-1.0, that would be great since the "full name" on the license list doesn't currently include the version number. Appreciate if you can submit it as a separate PR from this one so that we can track them separately. Thank you!

Reply:
You got it -- I'll send a follow-up PR.

Reply:
https://github.com/spdx/license-list-XML/pull/1279